### DISCUSSION TITLE
Would PotatoNV support Huawei K3V2 chipsets anytime soon?

### DISCUSSION THREAD

**Discussion by tmhakim** [kitsuned/PotatoNV#139] [Ideas]
Hello!

I was wondering that would PotatoNV support K3V2 chipsets in the future releases because i wanted to unlock my tablet (Mediapad 10 FHD) since i got it in Christmas 2012 but couldn't do so. now the tablet is very laggy in basic tasks and i wanted to run CyanogenMod 10 to atleast make the device usable and free up some memory.

I hope to get a reply soon!

Taher

**Comment by mashed-potatoes** [comment]
Hi Taher (@TAHERHAKIM),

Technically, PotatoNV is compatible with this legacy hardware, but you should not expect any guarantees.
You need to find the "board firmware" for this tablet or any other device on K3V2. This is quite difficult, since the SoC is already 10 years old. Most of the resources are dead.

But if you really find the archive with the board firmware — send it to me. Upload it somewhere and leave a link in this thread, or message me at the Telegram support group.

After I'll make a bootloader for you. If PotatoNV really works with K3V2, bootloader will be included in the next PotatoNV release.

Andrey.

**Reply by tmhakim** [reply]
Hello Andrey,

I am glad that you replied to my message.

Currently i am finding difficulty finding the board firmware for my model s10-101w. but i did find something called board software for my tablet but for the 3G/LTE version. i am currently having a WiFi-only model 

Q1. Is board firmware and board software the same thing?
Q2. If can't find my board firmware so can i use any K3V2 board firmware for my tablet?

**Reply by mashed-potatoes** [reply]
Hi Taher,

LTE model is fine if it uses K3V2.

> Q1. Is board firmware and board software the same thing?

Yes.

> Q2. If can't find my board firmware so can i use any K3V2 board firmware for my tablet?

Yes. You can pick any device. Even phone on K3V2.
